requirements.txt

```diff
@@ -70,6 +72,7 @@ mysqlclient==1.3.9
 netaddr==0.7.19
 netifaces==0.10.5
 oauthlib==2.0.0
+olefile==0.46
```

Comment:
@giovannicimolin, why is `olefile` needed for this PR? It appears to be totally unrelated.